### PR TITLE
JAMES-2148 create domain mappings via the CLI

### PR DESCRIPTION
**Pull request by Maxxx873**
JAMES-2148 create domain mappings via the CLI:

* added getDomainMappings method in RecipientRewriteTableManagementMBean and RecipientRewriteTableManagement;
* added the following commands in CmdType:
     - AddDomainMapping,
     - RemoveDomainMapping, 
     - ListDomainMappings;
* added the corresponding tests in CmdTypeTest;
* added the code for handling commands in ServerCmd;
* added the corresponding tests in ServerCmdTest

### CODE REVIEW

File: server/apps/cli/src/test/java/org/apache/james/cli/ServerCmdTest.java [hunk 1]
```diff
@@ -19,34 +19,14 @@
 
 package org.apache.james.cli;
 
-import static org.assertj.core.api.Assertions.assertThat;
```
Comment (chibenwa) [inline server/apps/cli/src/test/java/org/apache/james/cli/ServerCmdTest.java:22]: Those import reorderare not needed

**Review by chibenwa — APPROVED** [review]

**Review by Arsnael — APPROVED** [review]
Nice little contribution, thanks and welcome :)

**Comment by Maxxx873** [comment]
> Nice little contribution, thanks and welcome :)

I like this project and it's interesting to follow its development. I hope to contribute to it.

**Review by quantranhong1999 — APPROVED** [review]
Hi,
Thank you for your contribution <3
It would be even greater if you could add some docs in `src/site/markdown/server/manage-cli.md`.

**Comment by Maxxx873** [comment]
> Hi, Thank you for your contribution <3 It would be even greater if you could add some docs in `src/site/markdown/server/manage-cli.md`.

Thanks for the comment! I'll be sure to update the documentation a bit later.

**Review by chibenwa — APPROVED** [review]

**Review by Arsnael — APPROVED** [review]

**Comment by chibenwa** [comment]
Thank you for the contribution!

**Pull request merged**